.github/project.yml

```diff
@@ -1,4 +1,4 @@
 release:
   current-version: 1.1.0
-  next-version: 1.1.1-SNAPSHOT
+  next-version: 2.0.0-SNAPSHOT
```

Comment:
Please, don't change this file in a PR that is not meant to do a release.

Reply:
Done.